src/main/java/edu/byu/cs/util/RepoUrlValidator.java

```diff
@@ -2,23 +2,32 @@
 
 import edu.byu.cs.autograder.GradingException;
 import org.eclipse.jgit.annotations.Nullable;
+import org.eclipse.jgit.api.CloneCommand;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.File;
+import java.util.UUID;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class RepoUrlValidator {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RepoUrlValidator.class);
 
-    public static boolean isValid(@Nullable String repoUrl) {
-        try {
-            clean(repoUrl);
-            return true;
-        } catch (InvalidRepoUrlException e) {
+    public static boolean isValidRepoUrl(String url) {
```

Comment:
Renaming the static method from `RepoUrlValidator.clean()` to `RepoUrlValidator.cleanRepoUrl()` introduces redundancy and duplication in the naming. Additional justification should be provided to defend this as a good change.

Reply:
I'll push back only against the idea that "additional justification" is needed here.

It's just cause clean() and cleanRepoUrl() were merged, and that happened to be the one that was kept. If we think clean() is a bit more concise, then sure